setup.py

```diff
@@ -683,6 +725,24 @@ def get_snappy_clib(field=None):
     return config[field]
 
 
+def get_sperr_clib(field=None):
+    """sperr static lib build config"""
+    sperr_dir = "src/SPERR"
+
+    config = dict(
+        sources=glob(f"{sperr_dir}/src/*.cpp"),
```

Comment:
@shaomeng, this builds all the src/ files, is it OK?

Reply:
Yes, it should be building all the .cpp files.